front/app/containers/Admin/reporting/components/ReportBuilder/Editor/RenderNode/index.tsx

```diff
@@ -26,6 +26,15 @@ const StyledBox = styled(Box)`
         box-sizing: content-box;
       `
       : 'cursor: move;'}
+
+  ${({ outlineColor }) =>
```

Comment:
The changes are unrelated to the rest of the PR- I am just using an `outline` instead of a `border`, because an `outline` doesn't add extra pixels. This way the view is slightly more accurate